src/util/config/index.js

```diff
@@ -213,13 +216,24 @@ config.set(
 
 config.set('isProd', config.get('env') === 'production');
 config.set('isDev', config.get('env') === 'development');
-config.validate();
 const appConf = config.get('app_server');
 if (
 	appConf.protocol === 'https' &&
 	(!fs.existsSync(appConf.key_file) || !fs.existsSync(appConf.crt_file))
 ) {
-	throw new Error('Missing HTTPS cert or key!');
+	const message = 'Missing HTTPS cert or key for application server!';
+	if (config.isProd) {
```

Comment:
ah, yes - was just fixing that up. I'm going to keep the 'http in prod' behavior for now and wait for a separate bug to enable https behind the load balancer since it's not strictly necessary right now and I don't want to make this more complicated than necessary